apps/ehr/src/features/external-labs/components/details/Questionarie.tsx

```diff
@@ -71,17 +71,17 @@ export const Questionarie: React.FC = () => {
     return null;
   }
 
-  return (
-    taskStatus === 'pending' && (
-      <OrderCollection
-        aoe={aoe}
-        collectionInstructions={collectionInstructions}
-        specimen={specimen}
-        serviceRequestID={serviceRequestID}
-        serviceRequest={serviceRequest}
-        _onCollectionSubmit={handleSampleCollectionTaskChange}
-        oystehr={oystehrZambda}
-      />
-    )
+  return taskStatus === 'pending' ? (
```

Comment:
this file name has a typo

Reply:
Let's not rename files while we are working, or we may get more complicated conflicts. I also changed this component a bit already. I suggest leaving a TODO comment.